irispy/iris_tools.py

```diff
@@ -672,11 +890,13 @@ def calculate_photons_per_sec_to_radiance_factor(
         for input wavelengths.
 
     """
+    # Get best fit by the method of least squares
+    iris_fit = fit_iris_xput(time_obs, time_cal_coeffs, cal_coeffs)
     # Get effective area and interpolate to observed wavelength grid.
-    eff_area_interp = _get_interpolated_effective_area(detector_type, wavelength)
+    eff_area_interp = _get_interpolated_effective_area(iris_fit, detector_type, wavelength)
     # Return radiometric conversed data assuming input data is in units of photons/s.
-    return constants.h * constants.c / wavelength / u.photon / \
-           spectral_dispersion_per_pixel / eff_area_interp / solid_angle
+    return constants.h * constants.c / wavelength / u.photon / spectral_dispersion_per_pixel / eff_area_interp / solid_angle
+
 
 def _get_interpolated_effective_area(detector_type, obs_wavelength):
```

Comment:
Too much has been reverted in a recent commit.  Change lines 901 to 904 to:
```python
def _get_interpolated_effective_area(detector_type, obs_wavelength, **kwargs):
   """Put docstring back"""
    # Get effective area
    iris_response = get_iris_response(pre_launch=True)
```
The rest of the function remains untouched.

Reply:
Yup, done. Will work on an improvement of this docstring in the next PR... Will need to go through each and every docstring then update and expand on it where appropriate.